### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
# What?
This plugin will Capture all rich text fields i.e of media type `text/richtext` and transform it into `ContentfulRichText`.
After that, it will append a field `html` to it and set it's value to the html value of `documentToHtmlString` from `'@contentful/rich-text-html-renderer'`.
A user can pass all the rendering options that the library support through the plugin config.

- [x] Add DOCS to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-# gatsby-transformer-contentful-richtext
\ No newline at end of file
+# gatsby-transformer-contentful-richtext
+
+Parses Contentful Rich text schema
```
Comment (sbezludny) [inline README.md]:
```suggestion
Parses Contentful Rich Text document
```

**Review by sbezludny — COMMENTED** [review]
Great job!
1. Please add EOF where missing
2. Would be nice to see the examples, I guess it's in your todo (`Add DOCS to the README`)

The user of the plugin defines the renderers only once and therefore there's no way to redefine them on a later stage. 
e.g. I want to render the same document twice on the page but using different renderers

Do I understand correctly?

**Comment by Khaledgarbaya** [comment]
@sbezludny Yes, you can still get query the original content and render it using the rich-text-html-renderer manually in case of an advanced use case, also for embedded entries you can check for types and send back the right html string

**Comment by Khaledgarbaya** [comment]
Another thing, this PR is about the code and test and I will add an example project in a different PR

**Comment by toby-bushell** [comment]
Can this be merged?

**Comment by Khaledgarbaya** [comment]
@toby-bushell this PR is blocked so far because of this issue https://github.com/gatsbyjs/gatsby/issues/9488

**Pull request merged**